### PR TITLE
coco attestation database schema

### PR DESCRIPTION
**Pull request by mcalmer**
## What does this PR change?

Confidential Compute Attestation Feature

## GUI diff

No difference.

- [x] **DONE**

## Documentation
- Documentation issue was created: https://github.com/SUSE/spacewalk/issues/23652

- [x] **DONE**

## Test coverage
- No tests: already covered

- [x] **DONE**

## Links

Issue(s): https://github.com/SUSE/spacewalk/issues/23550

- [x] **DONE**

## Changelogs

Make sure the changelogs entries you are adding are compliant with https://github.com/uyuni-project/uyuni/wiki/Contributing#changelogs and https://github.com/uyuni-project/uyuni/wiki/Contributing#uyuni-projectuyuni-repository

If you don't need a changelog check, please mark this checkbox:

- [x] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

## Re-run a test

If you need to re-run a test, please mark the related checkbox, it will be unchecked automatically once it has re-run:

- [ ] Re-run test "changelog_test"
- [ ] Re-run test "backend_unittests_pgsql"
- [ ] Re-run test "java_pgsql_tests"
- [ ] Re-run test "schema_migration_test_pgsql"
- [ ] Re-run test "susemanager_unittests"
- [ ] Re-run test "javascript_lint"
- [ ] Re-run test "spacecmd_unittests"

# Before you merge

Check [How to branch and merge properly](https://github.com/uyuni-project/uyuni/wiki/How-to-branch-and-merge-properly)!

### CODE REVIEW

**Review by lucidd — APPROVED** [review]

**Comment by jordimassaguerpla** [comment]
Hi ! We fixed a spelling mistake in the name of the tests and you will have to rebase your PR for the correct tests to run. Sorry for the inconvenience.

**Comment by mcalmer** [comment]
@parlt91 no, please don't review. This is a collector PR for our feature. We will merge it before beta 2 and will get more commits.

**Review by rjmateus — APPROVED** [review]
lgtm

File: containers/server-attestation-image/server-attestation-image.changes [hunk 1]
```diff
@@ -0,0 +1,4 @@
+-------------------------------------------------------------------
+Thu Feb 22 09:44:29 UTC 2024 - Thomas Florio <thomas.florio@suse.com>
+
+- Initial version
```
Comment (deneb-alpha) [inline containers/server-attestation-image/server-attestation-image.changes:4]: missing  empty line
```suggestion
- Initial version

```

File: containers/server-attestation-image/tito.props [hunk 2]
```diff
@@ -0,0 +1,3 @@
+[buildconfig]
+tagger = tito.tagger.SUSEContainerTagger
+builder = custom.ContainerBuilder
```
Comment (deneb-alpha) [inline containers/server-attestation-image/tito.props:3]: missing empty line
```suggestion
builder = custom.ContainerBuilder

```

**Review by deneb-alpha — APPROVED** [review]
The PR  would require a deeper review but apparently I'm the blocker so approving with remarks.

**Review by mackdk — APPROVED** [review]

**Review by Etheryte — APPROVED** [review]

**Pull request merged**